indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -24,7 +24,13 @@
                             <th class="i-table">{% trans %}Full name{% endtrans %}</th>
 
                             {% for item in static_columns %}
-                                <th class="i-table" data-sorter="text">{{ item.caption }}</th>
+                                {% if not item.get('filter_only') %}
```

Comment:
this one can be inlined in the for loop (`{% for item in static_columns if not item.get('filter_only') %}`)